### PR TITLE
Fix cross-references

### PR DESCRIPTION
**Pull request by oscargus**
## PR Summary

Fixed even more missing cross-references. Thought it was time to also update the file containing the missing ones.

Seems like See also cannot use ~-references.

## PR Checklist

**Tests and Styling**
- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).

**Documentation**
- [ ] New features are documented, with examples if plot related.
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).

### CODE REVIEW

**Comment by oscargus** [comment]
Need to think through `input_dims` and `output_dims`.

File: examples/misc/ftface_props.py [hunk 1]
```diff
@@ -3,9 +3,9 @@
 Font properties
 ===============
 
-This example lists the attributes of an `.FT2Font` object, which describe
-global font properties.  For individual character metrics, use the `.Glyph`
-object, as returned by `.load_char`.
+This example lists the attributes of an ``FT2Font`` object, which describe
```
Comment (timhoffm) [inline examples/misc/ftface_props.py:6]: Changing references to literal is good for old changelogs when the referenced code is removed. In other cases, there are often better solutions. I'm away from my dev machine, so cannot easily check but I suspect that we have made these private. If so, we shouldn't mention them in the docs anymore at all.

File: lib/matplotlib/axes/_axes.py [hunk 2]
```diff
@@ -322,7 +322,7 @@ def inset_axes(self, bounds, *, transform=None, zorder=5, **kwargs):
             Lower-left corner of inset Axes, and its width and height.
 
         transform : `.Transform`
-            Defaults to `ax.transAxes`, i.e. the units of *rect* are in
+            Defaults to ``ax.transAxes``, i.e. the units of *rect* are in
```
Comment (timhoffm) [inline lib/matplotlib/axes/_axes.py:325]: We should have a documentation reference for this one way or the other. To be checked: why don't we have this in our api reference.

Same for some other cases below.

Comment (oscargus) [reply lib/matplotlib/axes/_axes.py:325]: I think the problem here is that `ax` is an object, so maybe one should refer to the class instead.

File: examples/misc/ftface_props.py (hunk 1 above)
Comment (oscargus) [reply examples/misc/ftface_props.py:6]: Not sure if they are private or if there is limited documentation for the classes that originates from C.

File: lib/matplotlib/axes/_base.py [hunk 3]
```diff
@@ -2481,7 +2481,7 @@ def relim(self, visible_only=False):
 
     def update_datalim(self, xys, updatex=True, updatey=True):
         """
-        Extend the `~.Axes.dataLim` Bbox to include the given points.
+        Extend the ``ax.dataLim`` Bbox to include the given points.
```
Comment (timhoffm) [inline lib/matplotlib/axes/_base.py:2484]: Ah, this is documented as an attribute to the class https://matplotlib.org/devdocs/api/axes_api.html#matplotlib.axes.Axes, but apparently one cannot link to these definition :(

File: examples/misc/ftface_props.py [hunk 4]
```diff
@@ -3,9 +3,9 @@
...
+global font properties.  For individual character metrics, use the ``Glyph``
+object, as returned by ``load_char``.
```
Comment (timhoffm) [inline examples/misc/ftface_props.py:8]:
```suggestion
object, as returned by ``FT2Font.load_char()``.
```
If we don't have a link, we must be a bit more explicit.

File: lib/matplotlib/axes/_axes.py (hunk 2 above)
Comment (timhoffm) [reply lib/matplotlib/axes/_axes.py:325]: The deeper problem is that `transAxes` is a simple attribute, which is not documented, and even if it was, we currently cannot reference it (c.f. `dataLim`).

Let's go for literal for the time being. I've opened #22897 to track this.

File: lib/matplotlib/axes/_base.py [hunk 5]
```diff
@@ -2222,7 +2222,7 @@ def add_artist(self, a):
 
     def add_child_axes(self, ax):
         """
-        Add an `.AxesBase` to the Axes' children; return the child Axes.
+        Add an ``AxesBase`` to the Axes' children; return the child Axes.
```
Comment (timhoffm) [inline lib/matplotlib/axes/_base.py:2225]:
```suggestion
        Add an Axes to this Axes' children; return the child Axes.
```
The exact type does not matter here.

File: examples/misc/ftface_props.py (hunk 1 above)
Comment (timhoffm) [reply examples/misc/ftface_props.py:6]: Ok, this is coming from C, and we never had documentation for that. Literal is good enough here.

File: lib/matplotlib/image.py [hunk 6]
```diff
@@ -62,8 +62,8 @@ def composite_images(images, renderer, magnification=1.0):
     Parameters
     ----------
     images : list of Images
-        Each must have a `make_image` method.  For each image,
-        `can_composite` should return `True`, though this is not
+        Each must have a ``make_image`` method.  For each image,
+        ``can_composite`` should return `True`, though this is not
```
Comment (timhoffm) [inline lib/matplotlib/image.py:66]:
```suggestion
        Each must have a ``make_image`` method.  For each image,
        ``can_composite()`` should return `True`, though this is not
```

**Comment by jklymak** [comment]
put to draft until revised...

**Comment by oscargus** [comment]
Closing this and may restart a boring day...

**Pull request closed without merging**